### PR TITLE
Fix OSX Clang

### PR DESCRIPTION
**Pull request by ax3l**
Try to fix the OSX Clang build.

Follow-up to #12 #19

- [x] ~~build number should have stayed > 1000 for new compilers, even with version bump~~ 0-based (now 2) fine again after compiler switch

### CODE REVIEW

**Comment by ax3l** [comment]
Interesting, it doesn't find ADIOS on cicleci now

**Comment by scopatz** [comment]
@conda-forge-admin, please rerender

File: recipe/cross-osx.cmake [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# this one is important
+set(CMAKE_SYSTEM_NAME Darwin)
+set(CMAKE_PLATFORM Darwin)
+# this one not so much
+set(CMAKE_SYSTEM_VERSION 1)
+
+# specify the cross compiler
+set(CMAKE_CXX_COMPILER $ENV{CC})
```
Comment (ax3l) [inline recipe/cross-osx.cmake]: typo: C

**Comment by ax3l** [comment]
Leaving this unmodified for documentation, fix in #20

**Pull request closed without merging**